network/firewall.go

```diff
@@ -248,3 +250,52 @@ func (c *FirewallClient) ListMachineRules(ctx context.Context, input *ListMachin
 
 	return result, nil
 }
+
+type ListRuleMachinesInput struct {
+	ID string
+}
+
+type Machine struct {
+	ID              string                 `json:"id"`
+	Name            string                 `json:"name"`
+	Type            string                 `json:"type"`
+	Brand           string                 `json:"brand"`
+	State           string                 `json:"state"`
+	Image           string                 `json:"image"`
+	Memory          int                    `json:"memory"`
+	Disk            int                    `json:"disk"`
+	Metadata        map[string]string      `json:"metadata"`
+	Tags            map[string]interface{} `json:"tags"`
+	Created         time.Time              `json:"created"`
+	Updated         time.Time              `json:"updated"`
+	Docker          bool                   `json:"docker"`
```

Comment:
Minor nit but I don't see this exercised through the new API's unit tests. See note below.